master/buildbot/test/unit/scripts/test_cleanupdb.py

```diff
@@ -36,6 +36,22 @@
 except ImportError:
     hasLz4 = False
 
+try:
+    import zstandard
+
+    [zstandard]
+    hasZstd = True
```

Comment:
snake_case for new code. I do understand that we use camelCase just a few lines above, but there's no need to make future cleanup larger.